src/modules/fancyzones/lib/FancyZones.h

```diff
@@ -54,6 +56,12 @@ interface __declspec(uuid("{2CB37E8F-87E6-4AEC-B4B2-E0FDC873343F}")) IFancyZones
      * Inform FancyZones that user has switched between virtual desktops.
      */
     IFACEMETHOD_(void, VirtualDesktopChanged)() = 0;
+    /**
+     * Callback from WinEventHook to FancyZones
+     *
+     * @param   data  Handle of window being moved or resized.
+     */
+    IFACEMETHOD_(void, HandleWinHookEvent)(const WinHookEvent* data){};
```

Comment:
We can clean up this interface now, `WindowCreated`, `MoveSize*` methods ar no longer called from DLL, but from within FZ lib.